### PR TITLE
Nest Troubleshooting info for "No Access to Partner Information" Error Update nest.markdown

### PR DESCRIPTION
**Pull request by genestealer**
Nest Troubleshooting infp for "No Access to Partner Information" Error Update nest.markdown

## Proposed change
This pull request adds new troubleshooting information to the documentation to help users resolve the "No access to partner information could not be retrieved" error message encountered during the setup wizard. The added content emphasizes the importance of migrating a Nest Account to a Google Account using the Google Nest App and provides guidance for situations where a home has multiple members. This information aims to improve the user experience and assist them in successfully linking to Home Assistant.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Example of error message:
image

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -466,6 +466,8 @@ Once you have completed the above steps, you can continue through the flow to re
 
 ## Troubleshooting
 
+- *No access to partner information* "Infomation could not be retrieved" message during the setup wizard, please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```
Comment (allenporter) [inline source/_integrations/nest.markdown]: I appreciate adding the documentation, thank you.

My impression is that this error message means that the account is not able to access the Home, and there are multiple reasons for this -- where the issue you'e cited here is probably one of the most common. I've made a slight adjustment to give a little more context.


```suggestion
- *No access to partner information* "Infomation could not be retrieved" message during the setup wizard means that the Google Account used is not able to access the Home.  Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```

We can also link to the nest docs here as well.
https://developers.google.com/nest/device-access/reference/errors/pcm#no_access_to_partner if that would be helpful.

**Review by allenporter — CHANGES_REQUESTED** [review]
Thank you for improving the documentation!  👍🏼

File: source/_integrations/nest.markdown (hunk 1 above)
Comment (genestealer) [reply source/_integrations/nest.markdown]: Sorry added a typo correction before seeing your suggestion. Will incorporate your edit now.

**Comment by genestealer** [comment]
@allenporter your suggested change has been incorporated, along with typo correction to my contribution.

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -466,6 +466,8 @@ Once you have completed the above steps, you can continue through the flow to re
 
 ## Troubleshooting
 
+- *No access to partner information* "Information could not be retrieved" error message during the setup wizard means that the Google Account used is not able to access the Home.  Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```
Comment (allenporter) [inline source/_integrations/nest.markdown]:
```suggestion
- *No access to partner information* "Information could not be retrieved" error message during the setup wizard means that the Google Account used is not able to access the Home. Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```

Comment (allenporter) [reply source/_integrations/nest.markdown]: I added an extra space.

**Review by allenporter — APPROVED** [review]

**Pull request merged**